File: app/src/main/java/com/babic/filip/movieshack/ui/list/MovieHolder.java
package com.babic.filip.movieshack.ui.list;

import android.support.v7.widget.RecyclerView;
import android.view.View;

import com.babic.filip.movieshack.model.Movie;

public abstract class MovieHolder extends RecyclerView.ViewHolder {

    public MovieHolder(final View itemView) {
        super(itemView);
    }

    abstract void displayItem(final Movie movie);
}

File: app/src/main/java/com/babic/filip/movieshack/ui/list/MovieHolder.kt
package com.babic.filip.movieshack.ui.list

import android.support.v7.widget.RecyclerView
import android.view.View

import com.babic.filip.movieshack.model.Movie

abstract class MovieHolder(itemView: View) : RecyclerView.ViewHolder(itemView) {

    abstract fun displayItem(movie: Movie)
}
